CONTRIBUTING.md

```diff
@@ -21,6 +21,23 @@ automatically during the first build.
 
 Please also note that the project requires at least Java 8 in order to build successfully. But we don't think this is really an issue nowadays.
 
+### Project Structure Overview
+
+The project comprises five primary Gradle modules:
+
+- **bigbone:** This module contains the core code for interacting with the Mastodon REST API, including tests.
+- **bigbone-rx:** This module serves as the proxy layer for forwarding requests to the main bigbone Gradle module.
```

Comment:
While it isn’t wrong that the Rx module “serves as the proxy layer”, I wouldn’t have worded it like this.

Maybe something like this?

```suggestion
- **bigbone-rx:** This module exposes methods available in the `bigbone` module as RxJava3 types so that you can use this library easily in an RxJava context.
```

Reply:
sorry, my bad. @Yordaniss `bigbone-rx` is actually calling the methods in the `bigbone` module (calls are forwarded 1:1), `bigbone-rx` thereof is sort of a thin layer on top of the `bigbone` and provides reactive streams functionality. Could you adopt the wording a bit into this direction? Thank you very much!

Reply:
@andregasser no problem. I would use than the suggestion from @PattaFeuFeu and combine this one with your feedback 🙂